### PR TITLE
default options in `package.json`

### LINKED ISSUE #500: default options in `package.json`
follow up for https://github.com/hoodiehq/hoodie/pull/490, I talked to @jameswestnz and we agreed it’d be a good idea to split up the pull request adding plugins into two:
1. Adding Hoodie options to `package.json`
2. Enabling plugins again :)

This issue is about step 1

### PR DESCRIPTION
**Pull request by gr2m**
fixes #500

### CODE REVIEW

File: server/config/defaults.js [hunk 1]
```diff
@@ -4,10 +4,8 @@ var path = require('path')
 
 function getDefaults () {
   var projectPath = process.cwd()
-  var pkg = require(path.join(projectPath, 'package.json'))
 
   return {
-    name: pkg.name,
```
Comment (gr2m) [inline server/config/defaults.js:10]: moved name into `app-options.js`

**Comment by gr2m** [comment]
ping @jameswestnz & @hoodiehq/maintainers for review

File: server/config/app-options.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+var join = require('path').join
+
+function getAppOptions () {
+  var projectPath = process.cwd()
+  var pkg = require(join(projectPath, 'package.json'))
+  var appOptions = pkg.hoodie || {}
+
+  appOptions.name = pkg.name
+
+  return appOptions || {}
```
Comment (HipsterBrown) [inline server/config/app-options.js]: Won't `appOptions` always exist based on line 8?

```
var appOptions = pkg.hoodie | | {}
```

Comment (gr2m) [inline server/config/app-options.js]: good catch, fixed

**Comment by rogeriochaves** [comment]
sounds good to me 👍

**Comment by rogeriochaves** [comment]
LGTM

**Comment by HipsterBrown** [comment]
Awesome! I'm stoked for the introduction of plugins again. Thanks for working on this @gr2m. LGTM 🚀

**Comment by gr2m** [comment]
@HipsterBrown It’s mostly @jameswestnz who does the heavy lifting right now :) But yeah, excited for plugins, too :)

**Comment by gr2m** [comment]
aaaaand it’s out: [v22.3.0](https://github.com/hoodiehq/hoodie/releases/tag/v22.3.0)

**Pull request merged**